pkg/config/config.go

```diff
@@ -67,8 +67,16 @@ const (
 	CertManagerCertificateClassName = "cert-manager.certificate.networking.knative.dev"
 
 	// ServingInternalCertName is the name of secret contains certificates in serving
-	// system namespace.
-	ServingInternalCertName = "knative-serving-certs"
+	// system namespace. (Deprecated, use ServingControlCertName or ServingRoutingCertName instead)
```

Comment:
Can we use `// Deprecated: NAME is deprecated.` as syntax? Also the comment on 71 seems redundant.